packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -815,4 +936,4 @@ class RenderEditable extends RenderBox {
       ),
     ];
   }
-}
+}
```

Comment:
nit: newline at EOF

Reply:
I don't think that's a new line? Looks like I just added it and deleted it.

Reply:
I mean - nit, add the newline at EOF :)